.github/workflows/build-release-artifacts.yml

```diff
@@ -0,0 +1,25 @@
+name: Build Release Artifacts
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - 'releases/**'
```

Comment:
I thought the release branches are named `*.release` in [create_releases.yml](https://github.com/firebase/firebase-android-sdk/blob/master/.github/workflows/create_releases.yml)?